AU2/plugins/util/WantedManager.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+
+from typing import Dict, Tuple, List
+
+from AU2.database.AssassinsDatabase import ASSASSINS_DATABASE
+from AU2.database.GenericStateDatabase import GENERIC_STATE_DATABASE
+from AU2.database.model import Event, Assassin
+
+
+class WantedManager:
+    """
+    Simple manager for wantedness
+    """
+
+    def __init__(self):
+        self.activated = GENERIC_STATE_DATABASE.plugin_map.get("WantedPlugin", False)
+        self.wanted_events: Dict[str, List[Tuple[datetime.datetime, Tuple[int, str, str]]]] = {}
+
+    def add_event(self, e: Event):
+        for playerID in e.pluginState.get("WantedPlugin", {}):
+            duration, crime, redemption = e.pluginState["WantedPlugin"][playerID]
+            expiry = e.datetime + datetime.timedelta(days=duration)
+            self.wanted_events.setdefault(playerID, [])
+            self.wanted_events[playerID].append((expiry, (duration, crime, redemption)))
+
+    def is_wanted_at(self, a: Assassin, date: datetime.datetime):
+        """
+        Returns true if:
+         1) the WantedPlugin is activated
+         2) `a` has a wanted event before `date`
+         2) `a`'s most recent wanted_event (at `date`) puts them wanted at `date`
+        """
+        if self.activated:
+            if not self.wanted_events.get(a.identifier, False):
```

Comment:
Consistency. Also, this should be a `defaultdict` so that you don't have to do this

```suggestion
            if not self.wanted_events.get(a.identifier, []):
```